Svg/Platform/Shared/WebRequestSvc.cs

```diff
@@ -31,7 +31,25 @@ internal string EnsureFullPath(string path)
             var filePath = path.StartsWith("file://") ? path.Substring(7) : path;
             if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
                 filePath = filePath.Trim('/');
-            return Path.IsPathRooted(filePath) ? filePath : Path.Combine(fs.GetDefaultStoragePath(), filePath);
+
+            // Path.Combine and Path.IsPathRooted do not work so we implemented them
```

Comment:
to not work where?
On Windows they do work.
Just not on android - maybe you can still preface that